pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.ruff]
-select = ["E", "F", "W", "C"]
-ignore = ["E501"]
+select = ["E", "F", "W", "C", "S"]
+ignore = ["E501", "S113"]
```

Comment:
it would be nice to have some comments telling us what rules are being selected/ignored and why(so the next person who comes in and wants to change the set has some context for why we have the existing set).  or at least link to the docs.

Reply:
> re E501 - imho we need to decide about the "right" max line length. I saw other PR that format for 80 columns which is (IMHO!) too small and the code is a bit ugly. OTOH now there are lines > 300 chars ;)

i would vote for at least 120.  Right now my IDE is showing about 140 chars wide.

> re S113 - we can add timeout, but I'm not sure about the right value (how fast/slow the server might be)

Note it as a TODO/followup in a comment in the file, that way at least we know why it's being ignored for now

Reply:
added, and I'll find the author (@vishnuchalla probably) to talk about the timeout